### DISCUSSION TITLE
Can't sign onto Mastodon, get an error message

### DISCUSSION THREAD

**Discussion by amgoth2000** [mastodon/mastodon#21990] [Q&A] [answered]
I've had a Mastodon account for several weeks and now when I try to go to my account I receive an error message: 
"404 Not Found
nginx/1.18.0"

Everything was working on 12/2/2022 but today Mastodon doesn't want to work and it won't even recognize my email.

I don't understand what happened and I don't know how to reach out to Mastodon to find out what happened.

Anyone know what I can do?

Thank you.

**Comment by mohe2015** [comment]
On which instance did this happen? It seems like the server administrator broke their server config.

**Reply by mohe2015** [reply]
Also as mastodon is federated you account always belongs to one server instance (your home server). So if you registered on `homeserver` your account is `@username@homeserver` which is different from `@username@anotherhomeserver`. This means you can only login on the instance you registered on.

**Reply by amgoth2000** [reply]
It happened today. I clicked on Mastodon and I get the error message I mentioned. Does this mean the server is permanently gone? Is that how Mastodon works? Sorry for all my questions, I've never had this issue with FB, IG, Twitter or ConscienceSocial so I am very confused. I really got used to Mastodon and rather miss it but if the server dies and that means I lose my account that means it can happen again and again right?

**Reply by mohe2015** [reply]
What do you mean with you clicked on "Mastodon". There is not "the" single server instance so it depends on which instance you were on.

**Reply by amgoth2000** [reply]
I clicked on the link I have for my account, just like I do for all my social media and I received the error message. The links always work (like for FB, twitter, IG, etc) but, today, it didn't work all morning.

But, good news, now the link works YAY! I guess the person running the server fixed the issue they were having.

But I guess this can happen again :(

**Reply by mohe2015** [reply]
> I clicked on the link I have for my account, just like I do for all my social media and I received the error message. The links always work (like for FB, twitter, IG, etc) but, today, it didn't work all morning.
> 
> But, good news, now the link works YAY! I guess the person running the server fixed the issue they were having.
> 
> But I guess this can happen again :(

I think I found your instance based on your twitter (sorry for "stalking"). It doesn't look too small so I assume they either just had some short issues which also occur to other social media from time to time or they may e.g. have updated their systems because of an ongoing attack (https://github.com/mastodon/mastodon/issues/21977)

**Reply by mohe2015** [reply]
Also you can follow cross instance etc. Mastodon consists of many servers and an important choice is which server you create your account on. Unfortunately currently this can also lead to loosing your account as far as I understand if the server gets unavailable. To my knowledge there is discussion about how to improve this situation (e.g. https://github.com/mastodon/mastodon/issues/19924)

**Reply by amgoth2000** [reply]
It's not stalking if you're being helpful lol I looked that 19924 and I'm afraid I didn't see any useful information but I'm not a techy person. Thank you for trying to help me though.

**Reply by mohe2015** [reply]
> It's not stalking if you're being helpful lol I looked that 19924 and I'm afraid I didn't see any useful information but I'm not a techy person. Thank you for trying to help me though.

Yeah there is not much *done* yet but they're discussing that issue. So maybe in the future there will be the possiblity to e.g. backup and move your account or have your account on multiple servers. Currently there's not much that can be done unfortunately. Except maybe making regular backups (there may be an option in the settings, not sure)

**Reply by DrWebX** [reply]
can't login at web site today...

**Reply by amgoth2000** [reply]
I am able to get on. I wonder if you're having the same server issue as I had the other day.

**Comment by nightpool** [comment, marked as answer]
@amgoth2000 @DrWebX 

This forum is for the development and maintenance of the *Mastodon server software*. This is a free software package that can be run by anyone who wants to provide their own, connected, federated social media software. It is not a website, just code that others use to run a website.

The people who maintain this software are, for the most part, not involved or associated with the individual websites that people run or sign up on. You'll have to contact the person running your server to get more details about what may or may not be happening here.

**Comment by DrWebX** [comment]
My Firefox needed permissions for add-ons, now says too many requests;)
